modules/apps/frontend-js/frontend-js-dependencies-web/npmscripts.config.js

```diff
@@ -35,6 +34,7 @@ module.exports = {
 			'graphiql/graphiql.css',
 			{name: 'axe-core', symbols: 'auto'},
 			{name: 'clipboard', symbols: 'auto'},
+			{name: 'cropperjs', symbols: 'auto'},
```

Comment:
LGTM as long as it works :sweat_smile:

Reply:
![](https://media0.giphy.com/media/TNO6mwK8s38vpHjh8Y/200.gif)